py/build.gradle

```diff
@@ -204,15 +199,13 @@ def copyH2OFromH2OWheel(String h2oPythonWheel) {
 task distPython(type: Zip, dependsOn: [checkPythonEnv, configurations.sparklingWaterAssemblyJar]) {
     doFirst {
         def H2O_HOME = System.getenv("H2O_HOME")
-        def H2O_PYTHON_WHEEL = System.getenv("H2O_PYTHON_WHEEL")
+        def H2O_PYTHON_WHEEL = wheelFile
 
         // if both properties are set, give precedence to H2O_HOME
-        if (H2O_HOME != null && H2O_PYTHON_WHEEL != null) {
-            copyH2OFromH2OHome(H2O_HOME)
-        } else if (H2O_HOME != null) {
+        if (H2O_HOME != null) {
             copyH2OFromH2OHome(H2O_HOME)
-        } else if (H2O_PYTHON_WHEEL != null) {
-            copyH2OFromH2OWheel(H2O_PYTHON_WHEEL)
+        } else {
+            copyH2OFromH2OWheel(H2O_PYTHON_WHEEL.toString())
```

Comment:
Is ```.toString()``` necessary here?

Reply:
Yup, `H2O_PYTHON_WHEEL` is `File`. I made it more obvious. thx!

Reply:
Actually no even here, we can pass in file directly